addons/knowledge/static/src/js/knowledge_views.js

```diff
@@ -12,6 +12,18 @@ const KnowledgeFormView = FormView.extend({
         Model: KnowledgeFormModel,
         Renderer: KnowledgeFormRenderer,
     }),
+    /**
+     * @override
+     * @param {Object} viewInfo
+     * @param {Object} params
+     */
+    init: function (viewInfo, params) {
+        params.withBreadcrumbs = false;
+        params.withControlPanel = false;
+        params.withSearchBar = false;
+        params.mode = 'readonly';
```

Comment:
Why do we want to open the form in read mode and not in edit mode ?
This problem is now that when we want to edit the title or the body it reloads the page completely, which is not really convenient
Also, This commit one could be merged with breadcrumb commit

Reply:
ok so we need to avoid the flickering and the reload of the side panel. But I assume that your rework of the form view will fix this, right ?